design-library/source/js/theme-scripts.js

```diff
@@ -0,0 +1,16 @@
+(function($) {
```

Comment:
I assume this file needs to go into Drupal? I can add it, just confirming it's not for pattern lab only